v2.1/Admin_manuals/Jetty9_Linux.adoc

```diff
@@ -0,0 +1,48 @@
+== Installation on Linux
+
+=== Prerequisites on Linux
+* Machine with at least 4 GB Ram (8GB are recommended, 16 GB and more if a large set of XML files are tested)
+* link:https://jdk.java.net/archive/[OpenJDK 11 is installed]
+* link:https://www.eclipse.org/jetty/download.php[Jetty 9 installed]
+
+=== Install ETF web application on Linux
+
+Download these resources provided in the ETF webpage:
+
+* link:https://github.com/etf-validator/etf-webapp/releases/tag/2.1.0-rc.1[etf-webapp.war] 
+* link:https://github.com/etf-validator/etf-suitd/releases/tag/2.1.0-rc.1[etf-suitd-legacy-2.1.0-SNAPSHOT.jar]
+* link:https://github.com/etf-validator/etf-tetd/releases/tag/1.1.0-rc.1[etf-tetd-1.1.0-SNAPSHOT.jar]
+* link:https://github.com/etf-validator/etf-bsxtd/releases/tag/3.0.0-rc.1[etf-bsxtd-legacy-2.1.0-SNAPSHOT.jar]
+
+Inside the ETF-webapp create a testdrivers folder, in which the  other three resources (etf-suitd-legacy-2.1.0-SNAPSHOT.jar, etf-tetd-1.1.0-SNAPSHOT.jar, etf-bsxtd-legacy-2.1.0-SNAPSHOT.jar) have to be added. 
+
+Download and include the following libraries inside etf-webapp.war/WEB-INF/lib/ to make the deployment work properly:
```

Comment:
As discussed in the last SG meeting this is a workaround that we should not publish in the documentation. Where should the administrator get the libraries from and is she/he really responsible for updating in case of security issues in these libraries...

Please add the libraries as dependencies to the SoapUI test driver. If you need assistance I am happy to help.